code/datums/crew_objective.dm

```diff
@@ -307,11 +330,161 @@ ABSTRACT_TYPE(/datum/objective/crew/janitor)
 					return 0
 			return 1
 
-//	bartender
+#define DRINK_OBJ_COUNT 3
+ABSTRACT_TYPE(/datum/objective/crew/bartender)
+/datum/objective/crew/bartender
+	shotgun
```

Comment:
ideally these would not be relatively pathed like this.

Reply:
fixed